test/functional/commands/find/by-css-e2e-specs.js

```diff
@@ -0,0 +1,78 @@
+import chai from 'chai';
```

Comment:
I like the tests. It would also make sense to add a documentation to this feature

Reply:
@mykola-mokhnach definitely. I'll add something to the appium docs and then probably an AppiumPro article.

Reply:
I'm going to do the iOS and Android-Espresso versions of this next. The Espresso version is going to be awesome... a hybrid of datamatchers and viewmatchers.

Reply:
for sure, if you want to guest write an appium pro article, i'd love to feature it